src/pages/documentation/migrate-to-swift.md

```diff
@@ -54,61 +54,9 @@ Save the `Podfile` and run `pod repo update` to update your local CocoaPods repo
 
 Once the previous command is complete, run `pod install` or `pod update` to update the application dependencies.
 
-### Swift Package Manager
+### Swift Package Manager (SPM)
```

Comment:
can we remove this file here and make this change in the acp repo (you will need to use the full link there instead of the local reference)?

Reply:
I will let @calebk1 handle this as part of his changes.